include/revng/Model/Importer/Binary/Options.h

```diff
@@ -0,0 +1,43 @@
+#pragma once
+
+//
+// This file is distributed under the MIT License. See LICENSE.md for details.
+//
+
+#include <cstdint>
+#include <string>
+
+#include "llvm/ADT/ArrayRef.h"
+#include "llvm/ADT/StringRef.h"
+#include "llvm/Support/CommandLine.h"
+
+// This interface description makes it hard to allow the user to specify
+// the value for the search depth with the desired granularity.
+//
+// For now, I'll set it to reasonable default value of `3`, which means that
+// the debug information is searched in the binary itself (level `1`), its
+// direct dependencies (level `2`) and the dependencies of those libraries.
+//
+// TODO: tweak the `Libraries` value / extend the enum if the need for more
+//       granular control ever arises.
+enum class DebugInfoLevel { No = 0, Yes = 1, Libraries = 3 };
+
+struct ImporterOptions {
+  const std::uint64_t BaseAddress;
+
+  const DebugInfoLevel DebugInfo;
+  const bool EnableRemoteDebugInfo;
+
+  const llvm::ArrayRef<std::string> AdditionalDebugInfoPaths;
```

Comment:
`ArrayRef` is non-owning, seems suspicious. You sure?

Reply:
`llvm::cl::opt` owns it, this way we don't pass a bunch of strings around - just distribute a pointer to a global.